app/services/geo_discovery/geoblacklight_document.rb

```diff
@@ -112,6 +113,10 @@ def private_references
         }
       end
 
+      def rights_boilerplate
```

Comment:
Is this used in this PR?

Reply:
Saw it come through unused in coverage - removed it, artifact of a previous implementation.